cypress/support/pages/confirmForm.pageObject.js

```diff
@@ -0,0 +1,27 @@
+import PageObject from '../PageObject';
+
+class ConfirmationFormPageObject extends PageObject {
+  url = '/cart.html';
+
+  get confirmWindow() {
+    return cy.get('.sweet-alert');
+  }
+
+  get customerName() {
+    return cy.get('.lead');
+  }
+
+  get confirmMsg() {
+    return cy.get('.sweet-alert > h2');
+  }
+
+  get customerCard() {
+    return cy.get('.lead');
+  }
+
+  get orderId() {
+    return cy.get('.lead');
```

Comment:
The selector `.lead` is used for `customerName`, `customerCard`, and `orderId`. Ensure that this selector uniquely identifies the intended element, or consider using more specific selectors to avoid potential conflicts.